assets/sass/components/_FrontPageContainer.scss

```diff
@@ -0,0 +1,51 @@
+.FrontPageContainer {
+  margin: 0;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 6em 0;
+  text-align: center;
+
+
+  .Title {
+    font-size: 2em;
+    margin: 0;
+    padding: 1.5em 0;
+
+    @media only screen and (max-width: 940px) {
```

Comment:
Are media queries really needed, as the layout did not change much.

Reply:
The logos on the front page are too small on mobile in my opinion. The motivation section has to be refactored anyway.

Reply:
Yeah, but introducing media queries for a temporary fix?

Reply:
For the logos, it's not a temporary fix. The logos have to be resized in mobile view.

It's just the _p_ tag that will be refactored as discussed in [this conversation](https://github.com/mib-unibe/unihack/pull/53#discussion_r249730347)

Reply:
Branche: 
![image](https://user-images.githubusercontent.com/1333699/51533146-512ba380-1e42-11e9-9d8f-103d83e9ba30.png)
iPhone X

![image](https://user-images.githubusercontent.com/1333699/51533306-aebff000-1e42-11e9-89dc-903b2610c55b.png)
iPhone 8

Master:

![image](https://user-images.githubusercontent.com/1333699/51533200-686a9100-1e42-11e9-9d38-5e623a33447e.png)
iPhone X Simulator.

![image](https://user-images.githubusercontent.com/1333699/51533332-c5fedd80-1e42-11e9-8f15-dd5f9a59eb5a.png)
iPhone 8

Reply:
There is no difference. Even on the iPhone 5s...

Reply:
Those changes are needed for they have finally the sponsors and organizers logos displayed on the index page.

The Logos have different spacing because they are to close to one another on the master branch.  The sizing compared to the desktop view is clearly different.

Reply:
Remove the media-query. I see the spacing differences but I am not convinced that a media-query is necessary.